mlrun/builder.py

```diff
@@ -51,18 +53,31 @@ def make_dockerfile(
     for build_arg_key, build_arg_value in build_args.items():
         dock += f"ARG {build_arg_key}={build_arg_value}\n"
 
-    if src_dir:
+    if source:
         dock += f"RUN mkdir -p {workdir}\n"
         dock += f"WORKDIR {workdir}\n"
-        dock += f"ADD {src_dir} {workdir}\n"
+        # ADD command automatically extracts compressed tar.gz files but not zip files
+        if source.endswith(".zip"):
```

Comment:
The comment is mismatched with the `if` statement. Do you mean: ?
```suggestion
        # ADD command automatically extracts compressed tar.gz files but not zip files
        if not source.endswith(".zip"):
```

Reply:
I mean that we need to extract the zip file ourselves because the ADD command doesn't extract zip files.
I'll rephrase the comment.